include/geode/basic/attribute_utils.h

```diff
@@ -195,4 +217,40 @@ namespace geode
     IMPLICIT_GENERIC_ATTRIBUTE_CONVERSION( unsigned int );
     IMPLICIT_GENERIC_ATTRIBUTE_CONVERSION( float );
     IMPLICIT_GENERIC_ATTRIBUTE_CONVERSION( double );
+
+#define IMPLICIT_ARRAY_GENERIC_ATTRIBUTE_CONVERSION( Type )                    \
+    template < size_t dimension >                                              \
```

Comment:
Is dimension a good name? I mean: it is maybe confusing with 2- or 3- dimension used for coordinates.

Reply:
I agree, maybe nb_items or another more explicit name ?